lib/ain-rs-exports/src/evm.rs

```diff
@@ -789,6 +789,60 @@ fn get_tx_by_hash(tx_hash: &str) -> Result<ffi::EVMTransaction> {
     Ok(out)
 }
 
+#[ffi_fallible]
+fn parse_tx_from_raw(raw_tx: &str) -> Result<ffi::EVMTransaction> {
+    let tx = SignedTx::try_from(raw_tx)?;
```

Comment:
Adds another conversion from raw_tx to SignedTx which is then repeated in `evm_try_unsafe_validate_transferdomain_tx_in_q` just after the call to this function. This check should either be performed there or the SignedTx cached.

Reply:
yes, @sieniven is working on cleaning this up.